IndomitableIbises/Bangazon-API-Sprint-1
# Delete & repopulate database with new table migrations and faker data

Made a file/command function that will delete and repopulate database with new table migrations and faker data

## Link to Ticket

No ticket to close

## Description of Proposed Changes

Made a file/command function that will delete and repopulate database with new table migrations and faker data

## Steps to Test

Only do this once you have data.sh installed into your machine root folder following joe's directions on 

```
https://github.com/nashville-software-school/bangazon-llc/blob/master/DB_RESET_SEED_SYSTEM.md
```

 Test:

```sh
git fetch --all
git checkout CR-Nuke-It-From-Space
```

1. Open your virtual environment and run:

``` 
pip install django_seed
```

2. Then inside the first layer of bangazon run : 

```
django_data.sh bangazon_api faker_factory
```

3. Once complete open database and make sure faker data has inputted along with new tables

## Impacted Areas in Application

List general components of the application that this PR will affect:

* sqlite database
* migrations

## Mentions @username

@rafcevallos 

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
